tests/dragonfly/snapshot_test.py

```diff
@@ -639,3 +639,34 @@ async def check_flag(key, flag):
 
     await check_flag("key1", 2)
     await check_flag("key2", 123456)
+
+
+@pytest.mark.parametrize("format", FILE_FORMATS)
+@dfly_args({**BASIC_ARGS})
+async def test_infomemory_for_stream(df_factory, format: str):
```

Comment:
why pytest and not unit test?

Reply:
you can also do dump and restore in the unit test, does not have to be a save

Reply:
also we can add checks to test test_rss_used_mem_gap
add a save and reload step there and
do all the memory accounting checks we have there after the reload